src/cogent3/core/new_sequence.py

```diff
@@ -171,7 +197,7 @@ def to_rich_dict(self, exclude_annotations=False):
         )
         if hasattr(self, "annotation_offset"):
             offset = int(self._seq.parent_start)
```

Comment:
can you just confirm that `parent_start` is on the SliceRecordABC

Reply:
yep